### PR TITLE
Notification Popout

### PR DESCRIPTION
**Pull request by zbarnz**
A notification component that you can pass text into

## **Preview:**

notification-cap

### CODE REVIEW

File: www/styles/Notification.module.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
+.container {
```
Comment (thudsonbu) [inline www/styles/Notification.module.css:1]: you will need to add `z-index` here in order to have the the notification bar appear above the page under any circumstance.

File: www/components/Notification.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styles from "../styles/Notification.module.css";
+import classNames from "classnames";
```
Comment (thudsonbu) [inline www/components/Notification.js:3]: Do we need to install this (aka add to `package.json`).

**Comment by thudsonbu** [comment]
Great job with the animations and keyframes they look great. Also clever use of `classnames`. 

Some feedback:
- I think we should move it to the top of the page so that the user can't miss it and it doesn't remind them of those crappy chatboxes
- From a style perspective we might wan't to add a box shadow around it to show that it is above the page (don't use black for the shadow)
- Since this is a notification we wan't it to be the the most vibrant color on the page so we will want our deepest blue.
- A feature that we will need immediately is the ability to specify notification type, such as error or success.

**Review by thudsonbu — CHANGES_REQUESTED** [review]
🏗️

**Comment by zbarnz** [comment]
> Great job with the animations and keyframes they look great. Also clever use of `classnames`.
> 
> Some feedback:
> 
> * I think we should move it to the top of the page so that the user can't miss it and it doesn't remind them of those crappy chatboxes
> * From a style perspective we might wan't to add a box shadow around it to show that it is above the page (don't use black for the shadow)
> * Since this is a notification we wan't it to be the the most vibrant color on the page so we will want our deepest blue.
> * A feature that we will need immediately is the ability to specify notification type, such as error or success.

okay did all this besides color, I tried a bunch of colors when making this and this one looked the best. If you can find something that looks better let me know.

**Review by Dahvd — APPROVED** [review]
nice

File: www/components/Notification.js (hunk 2 above)
Comment (zbarnz) [reply www/components/Notification.js:3]: No

**Pull request merged**